engines/strict_authentication/app/controllers/strict_authentication/authentication_controller.rb

```diff
@@ -35,7 +35,16 @@ def all_allowed_paths
       # so they can check if a customer has access to other products and show those
       # to them or verify paths
       all_product_versions = @system.products.map { |p| Product.where(identifier: p.identifier, arch: p.arch) }.flatten
-      all_product_versions.map { |prod| prod.repositories.pluck(:local_path) }.flatten
+      allowed_paths = all_product_versions.map { |prod| prod.repositories.pluck(:local_path) }.flatten
+      manager_prod = @system.products.any? { |p| p.identifier.include?('manager') }
```

Comment:
we should restrict this to SUMa Server. A proxy system woud gets it's cache from SUMa server and itself does not need access. If we just look for `manager` we enable a sneak through pass when someone has a proxy server running.

Reply:
That would be 
```
manager_prod = @system.products.any? { |p| p.identifier.downcase.include?('manager-server') }
```
then.